scripts/pin_the_slow_way.py

```diff
@@ -33,15 +33,15 @@
           'bzip2': 'bzip2 1.0.*',
           'fontconfig': 'fontconfig 2.11.*',
           'freetype': 'freetype 2.6.*',
-          'hdf5': 'hdf5 1.8.17',
+          'hdf5': 'hdf5 1.8.17|1.8.17.*',
           'icu': 'icu 56.*',
           'jpeg': 'jpeg 9*',
           'libnetcdf': 'libnetcdf 4.4.*',
           'libpng': 'libpng >=1.6.21,<1.7',
           'libtiff': 'libtiff 4.0.*',
-          'ncurses': 'ncurses 5.9.*',
+          'ncurses': 'ncurses 5.9*',
```

Comment:
Just a question: what's actually so bad about using the dot? Or does `5.9*` matches 5.10? Or is `.` interpreted as `anything` like in a regex?